wp-content/themes/pub/wporg-learn-2024/patterns/sensei-lesson-columns.php

```diff
@@ -29,7 +29,11 @@
 
 		<?php if ( $is_completed ) : ?>
 			<!-- wp:wporg/notice {"style":{"spacing":{"margin":{"top":"var:preset|spacing|20"}}}} -->
-			<div class="wp-block-wporg-notice is-tip-notice" style="margin-top:var(--wp--preset--spacing--20)">
+			<div class="wp-block-wporg-notice is-tip-notice" 
+				style="
+					margin-top: <?php echo $module ? 'var(--wp--preset--spacing--20)' : 'var(--wp--preset--spacing--0)'; ?>;
```

Comment:
If you need to unset this I'd use:
```suggestion
					margin-top: <?php echo $module ? 'var(--wp--preset--spacing--20)' : 'unset'; ?>;
```
Otherwise can you omit `margin-top` altogether?

Reply:
`margin-top` omitted. 
e49df94